lib/buildkite/test_collector.rb

```diff
@@ -23,7 +23,7 @@ class << self
     end
 
     def self.configure(hook:, token: nil, url: nil, debug_enabled: false, tracing_enabled: true)
-      self.api_token = token || ENV["BUILDKITE_ANALYTICS_TOKEN"]
+      self.api_token = token || ENV["BUILDKITE_ANALYTICS_TOKEN"]&.strip
```

Comment:
We probably want to strip both token passed from the `configure` call and `ENV`? 🤔